spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc

```diff
@@ -0,0 +1,212 @@
+* **Streams** Deploy/undeploy Stream Definitions
+* **Tasks** List, create, launch and destroy Task Definitions
+* **Jobs** Perform Batch Job related tasks
+* **Analytics** Create data visualizations for the various analytics modules
+
+Upon starting Spring Cloud Data Flow, the Dashboard is available at:
+
+"http://<adminHost>:<adminPort>/admin-ui"
+
+For example: http://localhost:9393/admin-ui
+
+If you have enabled https, then it will be located at https://localhost:9393/admin-ui
+
+If you have enabled security, a login form is available at http://localhost:9393/admin-ui/#/login
+
+**Note**: The default Dashboard server port is `9393`
+
+.The Spring Cloud Data Flow Dashboard
+image::images/dataflow-dashboard-about.png[The Spring Cloud Data Flow Dashboard, width=500]
+
+[[dashboard-apps]]
+== Apps
+
+The _Apps_ section of the Dashboard lists all the available applications and
+provides the control to register/unregister them (if applicable). By clicking on
+the magnifying glass, you will get a listing of available definition properties.
+
+.List of Available Applications
+image::images/dataflow-available-apps-list.png[List of available applications, width=500]
+
+[[dashboard-runtime]]
+== Runtime
+The _Runtime_ section of the Dashboard application shows the Spring Cloud Data Flow
+cluster view with the list of all running applications. For each runtime module the
+state of the deployment and the number of deployed instanced is shown.
+A list of the used deployment properties is available by clicking on the
+module id.
+
+.List of Running Applications
+image::images/dataflow-runtime.png[List of running applications, width=500]
+
+[[dashboard-streams]]
+== Streams
+
+The _Streams_ section of the Dashboard provides the _Definitions_ tab that provides
+a listing of Stream definitions. There you have the option to *deploy* or *undeploy*
+those streams. Additionally you can remove the definition by clicking on *destroy*.
+
+.List of Stream Definitions
+image::images/dataflow-streams-list-definitions.png[List of Stream Definitions, width=500]
+
+[[dashboard-tasks]]
+== Tasks
+
+The _Tasks_ section of the Dashboard currently has three tabs:
+
+* Modules
+* Definitions
+* Executions
+
+[[dashboard-tasks-modules]]
+=== Modules
+
+_Modules_ encapsulate a unit of work into a reusable component. Within the Dataflow
+runtime environment Modules allow users to create definitions for _Streams_ as
+well as _Tasks_. Consequently, the _Modules_ tab within the _Tasks_ section
+allows users to create _Task_ definitions.
+
+**Note**: You will also use this tab to create Batch Jobs.
+
+.List of Task Modules
+image::images/dataflow-task-modules-list.png[List of Task Modules, width=500]
+
+On this screen you can perform the following actions:
+
+* View details such as the task module options.
+* Create a Task Definition from the respective Module.
+
+==== Create a Job Definition from a selected Job Module
```

Comment:
Is this supposed to be `Create a Task Definition from a selected Task Module` instead of job?